client/commands/seeds/export.go

```diff
@@ -40,5 +44,20 @@ func exportSeed(cmd *cobra.Command, args []string) error {
 	}
 
 	// now get the output file and write it
-	return seed.Write(path)
+	return writeSeed(seed, path)
+}
+
+func writeSeed(seed certifiers.Seed, path string) (err error) {
+	var f *os.File
+	f, err = os.Create(path)
+	if err == nil {
+		stream := json.NewEncoder(f)
+		err = stream.Encode(seed)
+		f.Close()
```

Comment:
I still do like `	return errors.WithStack(err)`, which returns nil if err is nil, including that last if statement internally.

But maybe that is too much `pkg/errors` magic and an extra two lines doesn't hurt.